python/ray/data/tests/test_bulk_executor.py

```diff
@@ -127,7 +127,11 @@ def test_new_execution_backend_invocation(ray_start_10_cpus_shared):
     DatasetContext.get_current().new_execution_backend = True
```

Comment:
This test is called `test_bulk_executor` so shall we just force bulk execution mode instead?

Reply:
Makes sense!